step-templates/deploy-child-project.json

```diff
@@ -1,262 +1,273 @@
+      "DefaultValue": "N/A",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "5789f25c-6efb-4bbc-9236-2939faf7c553",
+      "Name": "ChildProject.ManualInterventions.UseApprovalsFromParent",
+      "Label": "Auto-Approve Child Project Manual Interventions",
+      "HelpText": "If the child project has manual interventions the step will look for manual interventions in the parent project.\n\nWhen a manual intervention in the parent project is found it will check that user's assigned teams.  If that user's assigned teams can approve the child project it will do so.\n\nPlease note, the user associated with the API key must be able to approve the child project manual interventions as well.  \n\nThe default is `Yes`, allow this to happen.  Set it to `No` to skip this functionality.",
+      "DefaultValue": "Yes",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Select",
+        "Octopus.SelectOptions": "No|No\nYes|Yes"
+      }
+    },
+    {
+      "Id": "2a93a2e9-562f-4985-8933-acbf1b0e6df6",
+      "Name": "ChildProject.ManualIntervention.EnvironmentToUse",
+      "Label": "Approval Environment",
+      "HelpText": "*Optional*\n\nThe name of the environment you wish to pull the approvals from for the parent project.   It will look at all the deployments for the current release of the parent project and select the latest deployment to the specified environment.\n\nUsed when you are deploying to `Production` but want to pull the approvals from `Staging` or a `Prod Approval` environment.\n\nDefaults to the current environment name.",
+      "DefaultValue": "#{Octopus.Environment.Name}",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "11101484-c68a-4a8b-aa4d-3f0a31e36ae2",
+      "Name": "ChildProject.ManualIntervention.Tenant.Name",
+      "Label": "Approval Tenant",
+      "HelpText": "*Optional*\n\nThe name of the tenant you wish to pull the approvals from for the parent project.   It will look at all the deployments for the current release of the parent project and select the latest deployment to the specified environment.\n\nUsed when you are deploying to `Production` but want to pull the approvals from `Staging` or a `Prod Approval` environment.\n\nDefaults to the current tenant name.  Will be skipped when doing a non-tenanted deployment.",
+      "DefaultValue": "#{Octopus.Deployment.Tenant.Name}",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    }
+  ],
+  "StepPackageId": "Octopus.Script",
+  "$Meta": {
+    "ExportedAt": "2023-10-27T22:53:45.921Z",
+    "OctopusVersion": "2023.2.13113",
+    "Type": "ActionTemplate"
+  },
+  "LastModifiedBy": "REOScotte",
+  "Category": "other"
```

Comment:
The category will need to be changed to `octopus`.

Reply:
This is resolved in the latest commit